action.yml

```diff
@@ -19,13 +19,10 @@ runs:
       shell: bash
     - run: pip install mypy
       shell: bash
-    - name: Install Dev Requirements
+    - name: Pip Install Local Package and All 'requirements*.txt' Files We Find
       run: |
-        ([ -e "requirements-dev.txt" ] && pip install -r requirements-dev.txt) || echo "no dev reqs"
-      shell: bash
-    - name: Pip Install Local Package (or All Requirements)
-      run: |
-        ([ -e "setup.py" ] && pip install .) || pip install `find . -name 'requirements.txt' | rev | sed -z 's/\n/ r- /g' | rev`
+        ([ -e "setup.py" ] && pip install .) || (echo "no setup.py")
+        pip install `find . -name 'requirements*.txt' | rev | sed -z 's/\n/ r- /g' | rev`
```

Comment:
does this need `-r`?

Reply:
the `-r` is included in that `sed` madness